opm/simulators/linalg/gpuistl/GpuSparseMatrix.cpp

```diff
@@ -81,6 +81,21 @@ GpuSparseMatrix<T>::GpuSparseMatrix(const T* nonZeroElements,
     }
 }
 
+template <class T>
+GpuSparseMatrix<T>::GpuSparseMatrix(const GpuVector<int> rowIndices,
```

Comment:
Why not a reference to `rowIndices` and `columnIndices`?

Reply:
good catch